pysat/utils/files.py

```diff
@@ -116,13 +116,17 @@ def process_parsed_filenames(stored, two_digit_year_break=None):
 
             return frame['files']
         else:
-            return pds.Series(files, index=index)
+            return pds.Series(files, index=index).sort_index()
     else:
         return pds.Series(None, dtype='object')
 
 
 def parse_fixed_width_filenames(files, format_str):
-    """Parse list of files, extracting data identified by format_str.
+    """Parse list of files, extracting data identified by `format_str`.
+
+    Uses the fixed-width format of the filename to find and parse
+    out variable information. The functions begins at the end of the filename,
+    parsing towards the begining.
```

Comment:
Recommend removing this, as I didn't understand what it meant until I read the 'Note' section.

Reply:
Removed.